packages/app-mobile/components/FolderPicker.tsx

```diff
@@ -0,0 +1,90 @@
+const React = require('react');
+
+import { FunctionComponent } from 'react';
+import { _ } from '@joplin/lib/locale';
+import Setting from '@joplin/lib/models/Setting';
+import Folder, { FolderEntityWithChildren } from '@joplin/lib/models/Folder';
+const { themeStyle } = require('./global-style.js');
+import Dropdown, { DropdownListItem, OnValueChangedListener } from './Dropdown';
+import { FolderEntity } from '@joplin/lib/services/database/types';
+
+interface FolderPickerProps {
+  disabled?: boolean;
+  selectedFolderId?: string;
+  onValueChange?: OnValueChangedListener;
+  mustSelect?: boolean;
+	folders: FolderEntity[];
+  placeholder?: string;
+  dark?: boolean;
+}
+
+
+const FolderPicker: FunctionComponent<FolderPickerProps> = ({
+	disabled,
+	selectedFolderId,
+	onValueChange,
+	mustSelect,
+	folders,
+	placeholder,
+	dark,
```

Comment:
We might be able to determine `dark` from just the `themeStyle(themeId)` (I think using `theme.appearance === ThemeAppearance.Dark`). As such, I don't think it needs to be a separate prop.

Given how this is being used in `folder.js`, I might be misunderstanding what `dark` is, though!

Reply:
Ah dark just means whether the text is dark (i.e. is it being displayed over a dark background). I'll update the prop to be more descriptive

Reply:
Fixed in c156b9c